README.md

```diff
@@ -645,7 +645,20 @@ Our [change log][changelog] has release history.
 implementation("com.squareup.okio:okio:2.4.1")
 ```
 
-Snapshot builds are [available][snap].
+<details>
+   <summary>Snapshot builds are also available</summary>
+      
+```kotlin
+repositories {
+   maven(url = "https://oss.sonatype.org/content/repositories/snapshots/")
```

Comment:
Does this syntax work in Groovy?

Reply:
Jep, you're right.
This was only possible with the Kotlin DSL.
I've just updated the syntax which works for both Gradle DSLs. See https://github.com/square/okio/pull/664/commits/69866a72b2fb9291177422d8cb38f3bebd8e4b1a